go.mod

```diff
@@ -3,3 +3,5 @@ module github.com/llir/irutil
 go 1.12
 
 require github.com/llir/llvm v0.3.0-pre7.0.20191220191840-fc6d0e8d9bc6
+
+replace github.com/llir/llvm => github.com/mewpull/llvm v0.3.0-pre7.0.20191221041541-3f58af5a0014
```

Comment:
Note, this PR depends on https://github.com/llir/llvm/pull/122. After llir/llvm#122 has been merged (if consensus is reached that it should be merged), then we will update `go.mod` to remove this replacement directive and use `llir/llvm` directly.

Reply:
> 

Note, this PR depends on llir/llvm#122. After llir/llvm#122 has been merged (if consensus is reached that it should be merged), then we will update go.mod to remove this replacement directive and use llir/llvm directly.

Done. Updated to use `llir/llvm`.